### DISCUSSION TITLE
Is there any way to update remote scripts automatically?

### DISCUSSION THREAD

**Discussion by SirNoSir** [evilmartians/lefthook#734] [Q&A] [answered]
Hello, We use the remotes option to centrally manage scripts. It has been very helpful. but when the remote scripts change,  I need to execute `lefthook install` to download the new scripts. Is there any way to avoid manual updates, such as downloading the config before each commit? Or Is there any plan to allow for remotes automatic updates?

**Comment by mrexox** [comment]
Hey! Right now lefthook does not download remotes on every call just to save time and resources and avoid redundant network usage. But if you find this useful you can suggest a featue.

For example, I can add a special configuration option like `refetch: true` to remotes configuration and it will trigger refetching of the code every time lefthook is called. Do you find this useful?

If so, please, submit a feature request. I'll be glad to help making remotes feature more useful

**Comment by SirNoSir** [comment]
Thank you for your response. I think this feature will be very useful and convenient when the scripts change frequently. I have sent a feature request #736 . Thanks again!

**Comment by mrexox** [comment, marked as answer]
Implemented `refetch: true` option: https://evilmartians.github.io/lefthook/configuration/refetch.html
